koku/masu/management/commands/migrate_trino_tables.py

```diff
@@ -339,51 +341,76 @@ def run_trino_sql(sql, schema=None) -> t.Optional[str]:
                 raise err
         except TrinoUserError as err:
             LOG.error(err.message)
-            return schema
+            return []
+
+
+def log_start(schemas: list[str]) -> str:
+    message = f"Running against the following schemas: {', '.join(schemas)}"
+    if (schema_count := len(schemas)) > 10:
+        message = f"Running against {schema_count} schemas"
```

Comment:
Why can't we just log the whole thing and then look in Kibana to see al the schema that are being modified?

Reply:
Are these logs being sent to Kibana? This is just a text string meant for human consumption so a long list of schemas is somewhat noisy. If we want to ship the full list of schemas to Kibana, that should be in a structured field that can be stored correctly in Elastic.